### DISCUSSION TITLE
About ruffle, full screen in a desktop PC.

### DISCUSSION THREAD

**Discussion by kanisuki** [ruffle-rs/ruffle#5571] [Q&A]
Please tell me in a desktop PC how to use ruffle in full screen.
I download the ruffle and double click the ruffle application.
It starts and I picked a swf with AS2, the swf shows the scene in 20 * 20 cm wide.
I make it in full screen, but in full screen only a quarter of the whole scene is shown.

**Comment by n0samu** [comment]
You might need to force the SWF to display with a different scale mode. Ruffle now allows you to select and force any scale mode from its "Open Advanced" menu (under File).
![image](https://github.com/ruffle-rs/ruffle/assets/71368227/651ead06-ac05-48f0-ab6e-93ca8d3c0903)
